apps/docs/src/.vitepress/components/ComponentRoadmap.vue

```diff
@@ -39,7 +34,7 @@ const usedStatus = computed(() => {
  * Components that should be displayed (considers the show all button).
  */
 const displayedComponents = computed(() => {
-  return sortedComponents.value.slice(0, showAll.value ? undefined : 24);
+  return sortedComponents.value.slice(0, showAll.value ? undefined : 16);
```

Comment:
Can we change the default value of `showAll` to `true` so the user sees all components initially and can then clic to show less? I already talked with Martin about this and its okay for him